tests/cypress/cypress.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Does this file need all these fields? Also the projectID seems a little random can this be more related to the project?
There also seems to be duplication of the iPhone viewport sizes and those are iPhone 5 sizes rather than iPhone4

Cleaner configuration file:

```
{
  "projectId": "nhsuk-frontend",
  "baseUrl": "http://localhost:3000",
  "chromeWebSecurity": false,
  "iPhone 5":
  {
     "viewportWidth": 320,
    "viewportHeight": 480
  },
  "baseUrl": "http://localhost:3000/components/",
  "liveUrl": "https://www.nhs.uk/",
}
```

Reply:
updated